pyproject.toml

```diff
@@ -36,6 +36,7 @@ keywords = [
 dependencies = [
     "pynwb>=2.8.0",
     "hdmf>=3.14.1",
+    "ndx_ophys_devices>=0.3.0",
```

Comment:
Note that this needs to be released first

Reply:
released! https://pypi.org/manage/project/ndx-ophys-devices/release/0.3.0/